docs/content/dagster-plus/insights.mdx

```diff
@@ -1,17 +1,17 @@
 ---
-title: "Dagster Cloud Insights | Dagster Docs"
+title: "Dagster+ Insights | Dagster Docs"
 description: "Visibility into historical usage and cost metrics."
 
 platform_type: "cloud"
 ---
 
-# Dagster Cloud Insights (Experimental)
+# Dagster+ Insights (Experimental)
 
 <Note>
   This feature is considered <strong>experimental</strong>.
 </Note>
 
-Using Dagster Cloud Insights, you can gain visibility into historical usage and cost metrics such as Dagster Cloud run duration, credit usage, and failures. You can also [integrate other metrics](#integrating-other-metrics) into the Insights UI and [export metrics from Dagster Cloud](#exporting-metrics).
+Using Dagster+ Insights, you can gain visibility into historical usage and cost metrics such as Dagster+ run duration, credit usage, and failures. You can also [integrate other metrics](#integrating-other-metrics) into the Insights UI and [export metrics from Dagster+](#exporting-metrics).
```

Comment:
This is a branding question, but seeing `Dagster+` everywhere feels a little clunky when it's not required to differentiate from OSS (e.g. "Dagster+ run duration")